src/main/java/org/ibp/api/java/impl/middleware/feedback/FeedbackServiceImpl.java

```diff
@@ -23,4 +27,14 @@ public void dontShowAgain(final FeedbackFeature feature) {
 		this.feedbackService.dontShowAgain(feature);
 	}
 
+	@Override
+	public FeedbackDto getFeedback(final FeedbackFeature feature) {
+		Optional<Feedback> feedback = this.feedbackService.getFeedback(feature);
+		if (feedback.isPresent()) {
+			return new FeedbackDto(feedback.get().getId(), feedback.get().getFeature(), feedback.get().getCollectorId(),
```

Comment:
@cuenyad you can implement a contructor in FeedbackDto that receives the Feedback entity to make it clear